src/zap/zapper.cpp

```diff
@@ -1394,6 +1394,14 @@ void Zapper::InitializeCompilerFlags(CORCOMPILE_VERSION_INFO * pVersionInfo)
 
 #endif // _TARGET_X86_
 
+#if defined(_TARGET_ARM64_)
+    static ConfigDWORD fFeatureSIMD;
+    if (fFeatureSIMD.val(CLRConfig::EXTERNAL_FeatureSIMD) != 0)
+    {
+        m_pOpt->m_compilerFlags.Set(CORJIT_FLAGS::CORJIT_FLAG_FEATURE_SIMD);
+    }
+#endif
+
```

Comment:
Why is this code needed? I may be mistaken, but I don't believe this will impact SIMD code generation, because the JIT will reject any pre-jitting of an method that has SIMD in it.

Reply:
@CarolEidt : @BruceForstall Suggested enabling SIMD code generation during crossgen since it is currently supported on all arm64 platforms.  This was my attempt to do that.  I thought this would be sufficient...

Reply:
I'm having some difficulty tracking down how code generation is disabled for `Vector<T>`. I thought there was a check in compiler.cpp, but can't find it. For x86/x64, it is important to not simply disable FeatureSIMD, but to ensure that no pre-compiled code is generated for a method that uses `Vector<T>` since its size is not known until runtime. I believe that check needs to be done in the JIT, so I was looking for where that happens.

Reply:
@CarolEidt 

So I just verified crossgen is generating SIMD enabled code for ARM64.  Code looks mostly identical except linkage related changes.

I think precompiling `Vector<T> ` is OK, because the importer replaces the SIMD Vector<T> method calls with intrinsics.  However, running x64 crossgen on a SIMD benchmark is probably a bad idea.  As I see do not see a trap on method that might have SIMD.  In fact since featureSIMD is disabled during crossgen it will not even be detected.

Reply:
Opened #15328

Reply:
I believe this discussion was resolved/closed in #15328